docs/details/base-widget/scroll.rst

```diff
@@ -139,23 +143,23 @@ Scroll momentum
 ---------------
 
 When the user scrolls a Widget and releases it, LVGL can emulate
-inertial momentum for the scrolling. It's like the Widget was thrown and
-scrolling slows down smoothly.
+inertial momentum for the scrolling. It's like the Widget was "thrown" (a
+:ref:`gesture <indev_gestures>`) and scrolling slows down smoothly.
```

Comment:
How gestures are related?

Reply:
> How gestures are related?

Ouch....  I guess gestures are not involved -- I was taking this knowledge from a dashboard project -- not LVGL.  My error.  Will correct.